presto-main/src/main/java/com/facebook/presto/operator/LookupJoinOperator.java

```diff
@@ -165,6 +171,8 @@ public Page getOutput()
                 if (!advanceProbePosition()) {
                     break;
                 }
+                statisticsCounter.recordProbe(joinSourcePositions);
```

Comment:
@idemura: You make a compelling argument, but given that there already exist a couple of join JMH benchmarks, getting some stats would make me more comfortable.